frontend/src/ches/MergeForm.jsx

```diff
@@ -22,7 +23,12 @@ const BODY_ENCODING = ['utf-8', 'base64', 'binary', 'hex'];
 const ATTACHMENT_ENCODING = ['base64', 'binary', 'hex'];
 
 const CONTEXTS_TYPES = ['xlsx', 'json'];
-const EXCEL_PARSING_DATENF = 'YYYY-MM-DD';
+// populate context/table formats
+const DT_FORMAT = 'YYYY-MM-DD';
```

Comment:
Since these are very generic format constants maybe worth refactoring to their own file at some point as they might be used elsewhere again.

Reply:
yeah, lots of refactoring possible.  i think i will tackle that when we add in a status tab CHES.

Reply:
Sounds good